sqlscripts/replication/0008-distributed-rand.sql

```diff
@@ -0,0 +1,3 @@
+CREATE TABLE IF NOT EXISTS %s -- operations table
+    ON CLUSTER '{cluster}' AS %s -- local operations table
+    ENGINE = Distributed('{cluster}', jaeger, %s, rand()); -- local operations table
```

Comment:
The same question as before the database name. What is the second parameter? If it is DB name then we should substitute it as well bc the DB name is configurable in the config.